pkg/kubelet/cm/dra/manager_test.go

```diff
@@ -750,27 +749,6 @@ func TestUnprepareResouces(t *testing.T) {
 					},
 				},
 			},
-			resourceClaim: &resourcev1alpha2.ResourceClaim{
-				ObjectMeta: metav1.ObjectMeta{
-					Name:      "another-claim-test",
-					Namespace: "test-namespace",
-					UID:       "test-reserved",
-				},
-				Spec: resourcev1alpha2.ResourceClaimSpec{
-					ResourceClassName: "test-class",
-				},
-				Status: resourcev1alpha2.ResourceClaimStatus{
-					DriverName: driverName,
-					Allocation: &resourcev1alpha2.AllocationResult{
-						ResourceHandles: []resourcev1alpha2.ResourceHandle{
-							{Data: "test-data", DriverName: driverName},
-						},
-					},
-					ReservedFor: []resourcev1alpha2.ResourceClaimConsumerReference{
-						{UID: "test-reserved"},
-					},
-				},
-			},
```

Comment:
Can you submit this as a separate PR? It's not related to this PR it seems.

Reply:
done in 8915d67676b